packages/cli-kit/src/output.ts

```diff
@@ -357,7 +357,7 @@ export function getOutputUpdateCLIReminder(
   const versionMessage = `💡 Version ${version} available!`
   if (!packageManager || packageManager === 'unknown') return versionMessage
 
-  const updateCommand = token.packagejsonScript(packageManager, 'shopify', 'upgrade')
+  const updateCommand = token.packagejsonScript(packageManager, 'shopify upgrade')
```

Comment:
This was showing something like `Run npm run shopify -- upgrade` for npm.

Reply:
If I'm not wrong I think that's necessary with `npm` because otherwise it treats those flags and arguments as `npm`'s

Reply:
Not sure what you mean. In this case, `upgrade` is not a flag, but part of the command, so we don't need the `--` separator. The third argument is used for flags, which do need the separator for npm.

We are doing the same in other places: https://github.com/Shopify/cli/blob/upgrade-cli2/packages/cli-hydrogen/src/cli/services/preview.ts#L23